src/azure.rs

```diff
@@ -74,6 +149,7 @@ impl From<Error> for super::Error {
 pub struct MicrosoftAzure {
     container_client: Arc<ContainerClient>,
     container_name: String,
+    blob_base_url: String,
```

Comment:
unfortunately the various blob clients do not expose public functions to generate the urls, so i needed to store it or use the most basic account client ...